_posts/installation/2019-03-15-oss-to-armory.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2019-03-13
+title: Upgrading from OSS to Armory Spinnaker
+categories:
+   - Installation
+description: Upgrading from OSS to Armory Spinnaker
+type: Document
+---
+
+If you are currently on OSS Spinnaker and interested in upgrading to Armory Spinnaker, you can easily upgrade if you used Halyard to install your Spinnaker cluster (you can also easily downgrade back to OSS Spinnaker)
+
+At a high level, the process to do this is as follows:
+* Back up your existing Halyard configuration
+* Switch from OSS Spinnaker to Armory Spinnaker
+* Update the Halconfig Spinnaker version from the OSS Spinnaker version to Armory Spinnaker
+* Deploy Armory Spinnaker, and remove any conflicting fields from your Halconfig
+
+Here are the actual steps:
+## Back up your existing Halyard configuration
+* From your existing Halyard, run `hal backup create`.  This will create a `tar` file; make sure you store this somewhere safe and external to your Halyard machine.
+
+## Switch from OSS Spinnaker Halyard to Armory Spinnaker Halyard
+
+* If you are using the OSS Halyard Docker image, run with something like this:
+
+    ```bash
+    docker run --name armory-halyard --rm \
+        -v ${PWD}/.hal:/home/spinnaker/.hal \
+        -v ${PWD}/.kube:/home/spinnaker/.kube \
+        -v ${PWD}/.secret:/home/spinnaker/.secret \
+        -it gcr.io/spinnaker-marketplace/halyard:stable
+    ```
+
+Then you can switch to Armory Halyard by just switching out the Docker image.  For example, as of 3/15/2019, the latest image is `docker.io/armory/halyard-armory:1.3.2` (you can recent/current image tags at https://hub.docker.com/r/armory/halyard-armory/tags).  Switch to the new image with this:
+
+    ```bash
+    docker run --name armory-halyard --rm \
+        -v ${PWD}/.hal:/home/spinnaker/.hal \
+        -v ${PWD}/.kube:/home/spinnaker/.kube \
+        -v ${PWD}/.secret:/home/spinnaker/.secret \
+        -it docker.io/armory/halyard-armory:1.3.2
+    ```
+
+* If you have Halyard directly installed on your machine, you can update to Armory Halyard with the Armory installer:
```

Comment:
Not sure if you've been recommending people to do this, but we've defaulted to just using the docker installation rather than macos. Not sure if anyone here runs macosx version